docs/products/tools/stackscripts/_index.md

```diff
@@ -6,28 +6,39 @@ tab_group_main:
-    product_description: "Use StackScripts to automatically configure new Linode instances using simple scripts. Create your own or browse the community StackScript library."
+    product_description: "Custom deployment scripts used to install software or configure a system on new Compute Instances."
+aliases: ['/platform/stackscripts-classic-manager/','/platform/stackscripts/','/platform/stackscripts-new-manager/','/stackscripts/','/guides/platform/stackscripts/']
 ---
 
-## Availability
+[StackScripts](http://linode.com/stackscripts/) provide Linode users with the ability to automate the deployment of custom systems. They work by running a custom script when deploying a new Compute Instance. These custom scripts store tasks that you may need to repeat often on new Compute Instances, such as:
 
-StackScripts can be created and deployed across [all regions](https://www.linode.com/global-infrastructure/).
+- Automating common system administration tasks, such as installing and configuring software, configuring system settings, adding limited user accounts, and more.
+- Running externally hosted deployment scripts.
+- Quickly creating Compute Instances for yourself or clients with the exact starter configuration you need.
+
+All StackScripts are stored in the Linode Cloud Manager and can be accessed whenever you deploy a Compute Instance. A StackScript authored by you is an *Account StackScript*. A *Community StackScript* is a StackScript created by a Linode community member that has made their StackScript publicly available.
 
 ## Features
 
 ### Quick and Easy Customization
 
-Find a StackScript, answer a few questions, and click to deploy. StackScripts run the first time your Linode instance is booted, allowing you to automatically customize the default Linux distribution.
+Select a StackScript, fill out any required fields, and click to deploy. StackScripts run the first time a Compute Instance boots, allowing you to automatically customize the default Linux distribution.
 
 ### A Library of StackScripts
 
-Customize your Linode with one of the many StackScripts in our library or discover a community-sourced script. They include everything from installing a Linode-optimized LAMP stack to configuring an IPsec VPN server.
+Customize your Compute Instance with one of the many StackScripts in our library or discover a community-sourced script. They include everything from installing a Linode-optimized LAMP stack to configuring an IPsec VPN server.
 
 ### Create Your Own
 
-Writing new StackScripts is simple. If you can’t find the right StackScript for your Linode, author your own using our [StackScripts guides](/docs/platform/stackscripts/).
+Writing new StackScripts is simple. If you can’t find the right StackScript for your needs, author your own.
+
+## Pricing and Availability
+
+StackScripts are available at no charge across [all regions](https://www.linode.com/global-infrastructure/).
 
-## Pricing
+## Technical Specifications
```

Comment:
Maybe here we should call out the different languages you could use? Seems like a good place to drop that info.

Reply:
Good idea. I added the following:

- Use any Linode-provided Linux distribution (see [Choosing a Linux Distribution](/docs/guides/choosing-a-distribution/))
- Supports the bash scripting language and any other scripting language supported by your chosen Linux distribution(s) (such as Python)

At some point, I think it may be worthwhile to add a guide on how to use various scripting languages with each of our popular distributions.